htdocs/compta/bank/line.php

```diff
@@ -130,9 +130,9 @@
 
 	$actarget = new Account($db);
 	if (GETPOST('accountid', 'int') > 0 && !$acline->rappro && !$acline->getVentilExportCompta()) {	// We ask to change bank account
-		$actarget->fetch(GETPOST('accountid', 'int'));
+		$account_id = $actarget->fetch(GETPOST('accountid', 'int'));
```

Comment:
This looks strange: A fetch return an error code (so 0 or -1) but you store it into a field that is an id of a bank account ?